operator/tests/e2e/e2e_test.go

```diff
@@ -89,7 +91,7 @@ var (
 type TestOptions struct {
 	OrganizationID     string
 	BillingAccountID   string
-	ServiceAccountID   string
+	serviceAccountID   string
```

Comment:
I assume this is because the variable is not set anywhere?  That seems like an over-zealous linter...